SOT-167: Add STARK mechanics document (#74)
sota-zk-labs/zkp-documents: terms/computational_integrity.md

---
comments: true
---

# Computational Integrity

> A proof system is said to be computational integrity (CI) when its claim is concern about the integrity of a particular computation.

Reviewer: ```suggestion
A proof system is said to ensure computational integrity (CI) when its claim is concerned with the integrity of a particular computation.
```
